### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by baha00** [matthew-kyles/CWA_Password_Extractor#1]
They changed path now... Need to loop to every client id

**Comment by matthew-hoad** [comment]
Hi baha00. Can you provide more information please?

**Comment by AlphaTJH** [comment]
I'm also having issues getting this to run, I'm not a coder by any means, but know how to read some code and make changes. 

The first issue I ran into was the args for the BaseURL, which in your readme state should be entered as 
`--base_url 'https://myorgname.hostedrmm.com'`

However, When I enter the URL as such, I get an error. I have to remove the single quotes around the URL for it to take,

Secondly, when I enter the baseURL this way, I get the traceback error below.

```
Traceback (most recent call last):
  File "C:\Users\REDACTED\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\requests\models.py", line 972, in json
    return complexjson.loads(self.text, **kwargs)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1520.0_x64__qbz5n2kfra8p0\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1520.0_x64__qbz5n2kfra8p0\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1520.0_x64__qbz5n2kfra8p0\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\REDACTED\Downloads\CWA_Password_Extractor-master\CW_Automate_PW_Extractor.py", line 22, in <module>
    cwclientids = pd.DataFrame(cwclientids.json())
  File "C:\Users\REDACTED\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\requests\models.py", line 976, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Unfortunately, I am not sure what the errors here are.  I did reach out to one of my friends that is familiar with Python and he stated that when he tries to run this using my bearer token he is getting '401 unautorized' as response from request made on line 22.

This happens even when using the admin credentials for Automate to generate our bearer token.

**Comment by baha00** [comment]
@matthew-kyles  sure with dual Berettas... On csgo :) bilbo...

@AlphaTJH  I got it working... Use python 3.8 thats first 
Base url is your company cw automate webpage... Not necessarily hostedrmm..

**Comment by AlphaTJH** [comment]
Switched to Python 3.8 as suggested by Baha, however, I am still getting the same traceback error as above (Same lines and everything).

Our baseURL is a hostedrmm.com URL, But thank you for mentioning that.

**Comment by AlphaTJH** [comment]
Just figured out how to get it working. For some context, I am running Python via Windows 11 Terminal, as I mentioned, I am not a python dev, so this is the easiest way for me to run python scripts.

The syntax requirements for this are different.  Out of sheer frustration, I started trying different ways to enter the args, and this is what is needed when running it the way I am

```
python CW_Automate_PW_Extractor.py --clientid "00000000-0000-0000-0000-000000000000" --bearer_token "bearer asdf1234asdf1234asdf1234==" --base_url "https://myorgname.hostedrmm.com" --output_file "cwa_passwords.csv"
```

You will notice I had to use double quotes around all values. Once I ran the script this way, it was able to process the password exports, and now have one very pretty CSV File.

Appreciate the input @baha00. 

Thanks much @matthew-kyles for creating this wonderful python script that just saved me another 4 or 5 days of manually exporting passwords.
